packages/manager/modules/manager-pci-common/src/components/deployment-mode-card/DeploymentModeCard.tsx

```diff
@@ -0,0 +1,110 @@
+import { useTranslation } from 'react-i18next';
+import { OsdsChip, OsdsText, OsdsTile } from '@ovhcloud/ods-components/react';
+import {
+  ODS_THEME_COLOR_INTENT,
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import { ODS_CHIP_SIZE } from '@ovhcloud/ods-components';
+import { clsx } from 'clsx';
+import './style.scss';
+import '@/translations/deployment-mode';
+import { ReactNode } from 'react';
+import { RegionChipByType } from '@/components/region-selector/RegionChipByType';
+
+export type TDeployment = {
+  name: string;
+  beta?: boolean;
+  comingSoon?: boolean;
+  leastPrice?: ReactNode;
+};
+
+export type DeploymentModeCardProps = Readonly<
+  {
+    labelId?: string;
+    ariaDetailsId?: string;
+  } & TDeployment
```

Comment:
`Readonly` marker has no effect on `TDeployment` object as it's not recursive

Reply:
AFAIS, `leastPrice ` is a React Node that should be recursively marked as `readonly`